cypher/cypher-docs/src/docs/dev/ql/administration/security/database/admin-database-syntax.asciidoc

```diff
@@ -5,74 +5,74 @@
 
 | [source, cypher]
 GRANT ACCESS
-    ON DATABASE name
+    ON DATABASE[S] {name \| *}
     TO role
```

Comment:
Maybe add `[, role2, ...]` to all the role parts to show they can be multiple

Reply:
I don't think it can be multiple roles in this place right? I thought that was only for `GRANT ROLE r1, r2 TO USER u1. u2. u3`. Or have we added this functionality recently?

Reply:
I don't think it is recent, but I'm pretty sure you can...

One of the parser tests: 
```
test(s"$command $privilege ON DATABASE * $preposition role1, role2") {
     yields(privilegeFunc(action, ast.AllGraphsScope() _, Seq("role1", "role2")))
}
```